vaadin-combo-box-flow-parent/vaadin-combo-box-flow/src/main/java/com/vaadin/flow/component/combobox/MultiSelectComboBoxSelectionModel.java

```diff
@@ -0,0 +1,130 @@
+package com.vaadin.flow.component.combobox;
```

Comment:
Let's add the license header to this file.

Reply:
Thanks, done.